guess-the-number/guess_the_number/guess.py

```diff
@@ -137,10 +144,39 @@ async def start_new_game(q: Q):
     await q.page.save()
 
 
+async def toggle_theme(q):
+
+    if not q.client.initialized:
+        q.page['meta_theme'] = ui.meta_card(box='', layouts=[
+            ui.layout(breakpoint='xs', zones=zones),
+            ui.layout(breakpoint='m', zones=zones),
+            ui.layout(breakpoint='xl', zones=zones),
+        ])
+        q.client.active_theme = 'default'
+        q.page['title_theme'] = ui.section_card(
+            box='title',
+            title='Plot theme demo',
+            subtitle='Toggle theme to see default plot colors change!',
+            items=[ui.toggle(name='toggle_theme',
+                             label='Dark theme', trigger=True)],
+        )
```

Comment:
Section card may not be the best fit here. Let's add the toggle to the header itself. As a bonus, you can also move the player name there so that it's nicer than the current state.